src/test/scala/pt/tecnico/dsi/openstack/designate/RecordsetsSpec.scala

```diff
@@ -0,0 +1,71 @@
+package pt.tecnico.dsi.openstack.designate
+
+import scala.util.Random
+import cats.effect.IO
+import pt.tecnico.dsi.openstack.common.models.WithId
+import pt.tecnico.dsi.openstack.designate.models.{Recordset, Zone}
+import pt.tecnico.dsi.openstack.designate.services.Recordsets
+
+class RecordsetsSpec extends Utils {
+  val withStubRecord: IO[(Recordsets[IO], String, Recordset.Create, WithId[Recordset])] =
+    for {
+      designate <- client
+      dummyZone <- designate.zones.create(Zone.Create("recordsets.org.", "john.doe@recordsets.org"))
+      recordsets = designate.zones.recordsets(dummyZone.id)
+      recordsetCreate = Recordset.Create(
+        name = s"sudomain${Random.nextInt()}.recordsets.org.",
```

Comment:
Codacy found an issue: [Don't use a predictable random number generator](https://app.codacy.com/gh/ist-dsi/scala-designate-client/pullRequest?prid=5821613)

Reply:
This is in tests, not really a problem.